client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -508,7 +508,7 @@ function StatsCommercialFlowOptOutForm( {
 				{ ( ! supportsOnDemandCommercialClassification ||
 					! isCommercial ||
 					( ! isClassificationInProgress && commercialClassificationLastRunAt > 0 ) ) && (
-					<Button variant="secondary" disabled={ isFormSubmissionDisabled } onClick={ formHandler }>
+					<Button variant="secondary" onClick={ formHandler }>
```

Comment:
```suggestion
					<Button variant="secondary" disabled={ ! isCommercial && isFormSubmissionDisabled } onClick={ formHandler }>
```

The user still needs to check all boxes to **_continue_** directing to the Stats Personal purchase, isn't it?

Reply:
oops good catch!

Reply:
it might make sense to separate it into two buttons

Reply:
> it might make sense to separate it into two buttons

I totally agree with it!

Reply:
Addressed 😄